packages/arb-token-bridge-ui/src/components/TopNavBar.tsx

```diff
@@ -5,6 +5,11 @@ import { twMerge } from 'tailwind-merge'
 import Image from 'next/image'
 import { useTransactionReminderInfo } from './TransactionHistory/useTransactionReminderInfo'
 
+export enum TabParamEnum {
+  BRIDGE = 0,
+  TX_HISTORY = 1
+}
```

Comment:
I would use string here instead, it's more robust if we ever change

Reply:
the query param is a string, but we need the numbers for the tab index